x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.ts

```diff
@@ -61,6 +61,12 @@ export class EnterpriseSearchRequestHandler {
     this.enterpriseSearchUrl = config.host as string;
   }
 
+  getBodyAsString(request: KibanaRequest<unknown, unknown, unknown>): string | undefined {
```

Comment:
Organization/dev experience request: I'd like to move this below `createRequest()` to its own section (defined by a comment block). In general, `createRequest` is the primary method of this class so it makes sense for it to come first and for devs to be able to quickly find/scroll it ASAP rather than having to scroll past a bunch of helpers first.

Once we add another request helper, I think it makes sense to keep those together conceptually as well.

Location-wise, I'm thinking maybe something like this?

```js
export class EnterpriseSearchRequestHandler {
  // ..

  createRequest() { ... }

  encodePathParams() { ... }

  /**
   * Body helpers
   */

  getBodyAsString() { ... }

  skipBodyValidation() { ... }

  /**
   * Response helpers
   */

  getErrorResponseBody() { ... }

  // ..etc
```

Reply:
FWIW, i did not add `skipBodyValidation` to this helper. This helper is specifically related to the request handler ... which is the second param of the route definition. And this file is already long, so I created a new helper.

Reply:
c0753d1122545d66ea064595bc4d5cb05815bac1

Reply:
Also reorganized 8fa74b89d11e1d8139b8d18f91ea8c0760c54066